pulumi/__main__.py

```diff
@@ -3,22 +3,35 @@
 from infra.autotag import register_auto_tags
 from infra.bucket import Bucket
 from infra.config import DEPLOYMENT_NAME, LOCAL_GRAPL
+from infra.dgraph_cluster import DgraphCluster, LocalStandInDgraphCluster
 from infra.dgraph_ttl import DGraphTTL
 from infra.engagement_creator import EngagementCreator
 from infra.metric_forwarder import MetricForwarder
 from infra.network import Network
 from infra.secret import JWTSecret
 from infra.service_queue import ServiceQueue
 
-if __name__ == "__main__":
 
+def _get_dgraph_cluster(network: Network) -> DgraphCluster:
```

Comment:
For overall clarity, I'd recommend calling this something like `_create_dgraph_cluster`. In Pulumi, getter functions are pretty exclusively associated with retrieving a reference to already existing infrastructure, which this is definitely not doing.